tests/workflow_management.test.ts

```diff
@@ -625,6 +626,22 @@ describe('test-list-steps', () => {
       return Promise.resolve(DBOS.workflowID);
     }
 
+    @DBOS.transaction()
+    static async transaction() {
+      return Promise.resolve(DBOS.workflowID);
+    }
+
+    @DBOS.transaction()
+    static async transactionWithError() {
+      await Promise.resolve();
+      throw Error('transaction error');
+    }
+
+    @DBOS.transaction({ readOnly: true })
```

Comment:
We're deprecating readOnly transactions.

```suggestion
    @DBOS.transaction()
```

Reply:
Actually, is this readOnlyTransaction used anywhere?

Reply:
With flush buffers , it was added to test the code path for flushBuffers.

But since flushBuffers is removed, this is no longer relevant

Reply:
in any case, removed